packages/themes/magister/icons.ts

```diff
@@ -11,7 +11,7 @@ export const icons = {
     description: 'icon.core.chevron-down',
     svg: '<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg"><path d="M239 401c9.4 9.4 24.6 9.4 33.9 0L465 209c9.4-9.4 9.4-24.6 0-33.9s-24.6-9.4-33.9 0l-175 175L81 175c-9.4-9.4-24.6-9.4-33.9 0s-9.4 24.6 0 33.9L239 401z" fill="var(--fill-default)"></path></svg>'
   },
-  circle: { svg: '<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8"/></svg>' },
+  circle: { svg: '<svg/>' },
```

Comment:
i don't think this is correct

Reply:
No idea. Perhaps something was changed in Figma? I'll see if I can revert it.

Reply:
Done